apps/site/pages/docs/building-sections/component-customization-overview.mdx

```diff
@@ -7,7 +7,7 @@ import { Callout } from 'nextra-theme-docs'
 
 <header>
 
-# Component Customization
+# Component & Section Customizations
```

Comment:
@renatamottam, since this is an overview doc for the section `Building and Customizing sections`, I believe we can keep the title as `Building and Customizing Sections`. This title effectively reflects the content's purpose as an overview. :)